pkgs/by-name/pa/panvimdoc/package.nix

```
@@ -0,0 +1,34 @@
+{ lib, fetchFromGitHub, stdenv, pandoc, vim, neovim-unwrapped, makeWrapper }:
+
+stdenv.mkDerivation (finalAttrs: {
+
+  pname = "panvimdoc";
+  version = "4.0.1";
+
+  src = fetchFromGitHub {
+    owner = "kdheepak";
+    repo = "panvimdoc";
+    rev = "v${finalAttrs.version}";
```

```suggestion
    rev = "refs/tags/v${finalAttrs.version}";
```

to avoid branch/tag ambiguity?